ztp/siteconfig-generator/siteConfig/siteConfigBuilder.go

```diff
@@ -193,21 +191,41 @@ func (scbuilder *SiteConfigBuilder) getExtraManifest(dataMap map[string]interfac
 				dataMap[filename] = value
 			}
 		} else {
-			manifestFile, err := scbuilder.fHandler.ReadSourceFile(filePath)
+			manifestFile, err := ReadFile(filePath)
 			if err != nil {
 				return dataMap, err
 			}
 			manifestFileStr := string(manifestFile)
 			dataMap[file.Name()] = manifestFileStr
 		}
 	}
+
+	// Adding End User Extra-manifest
+	files, err = GetFiles(clusterSpec.ExtraManifestPath)
+	if err != nil {
+		return dataMap, err
+	}
+	for _, file := range files {
+		if file.IsDir() {
+			continue
```

Comment:
Should we recurse into subdirectories?

Reply:
I will document this it needs to be flat dir for now then create enhancement as we discuss overlay & template for extra-manifest.